### ISSUE TITLE
Install target puts site-packages directory into wrong location

### ISSUE THREAD

**Issue by marcpaterno** [fnalacceleratormodeling/synergia2#14]
When configured with `-DCMAKE_INSTALL_PREFIX=${INSTALL_DIR}` the installation target ends up creating the `site-packages` directory at:

```
${INSTALL_DIR}/lib/python./site-packages/
```

The Python "major.minor" values are missing; in the case of my build, using Python 3.9.12 from Homebrew, the directory name should have been:
```
${INSTALL_DIR}/lib/python3.9/site-packages/
```

**Comment by egstern** [comment]
I see this also on ubuntu 22.04 which was using python3.10.

**Comment by marcpaterno** [comment]
Does the fix on the 14-install-target-puts-site-packages-directory-into-wrong-location branch fix this issue on Ubuntu 22.04?

**Comment by marcpaterno** [comment]
A problem caused by the Homebrew update of mpi4py is also fixed with this pull request.
